lib/drupal.js

```diff
@@ -3,7 +3,7 @@ import { DrupalClient } from "next-drupal";
 export const Drupal =
   typeof window !== "undefined"
     ? null
-    : new DrupalClient(process.env.NEXT_PUBLIC_DRUPAL_BASE_URL, {
+    : new DrupalClient(process.env.NEXT_PUBLIC_DRUPAL_BASE_URL?.replace(/\/$/, ""), {
```

Comment:
I get the sense this likely fixes our issue of the trailing base_url! Thank you!